CIP-0001/README.md

```diff
@@ -388,6 +411,12 @@ Current editors are listed here below:
 [@Crypto2099]: https://github.com/Crypto2099
 [@perturbing]: https://github.com/perturbing
 
+Emeritus editors:
```

Comment:
Very nice idea, love it!

Reply:
thanks & I believe it's also a way to pre-empt future issues like the one that developed here:
- https://github.com/cardano-foundation/CIPs/pull/366#pullrequestreview-1170652027

... for which I had suggested this as a solution to the deadlock we had over it: https://github.com/cardano-foundation/CIPs/pull/366#issuecomment-1308482496